### PR TITLE
Enabled the SCC CCIA roles, and renamed the FIB to the BSSB

### PR DESCRIPTION
**Pull request by Generalcamo**
This was a bit overdue, but it's been done now. No user facing changes for the normal player, but will probably need a review by CCIA and administrative staff.

### CODE REVIEW

**Comment by Generalcamo** [comment]
!review

**Review by Arrow768 — DISMISSED** [review]
You have updated various names.

The paths must be updated to reflect the new names.

If a item uses a icon with the old name as (or within the) icon name that should be updated (after a usage check to ensure the icon name isn't used somewhere else)

**Comment by Generalcamo** [comment]
It has been requested by Bear of the CCIA team that the old FIB items get resprited to conform to modern standards. As such, I'll be marking this WIP.

**Comment by Generalcamo** [comment]
!wip

**Comment by NonQueueingMatt** [comment]
Stale

**Comment by FluffyGhoster** [comment]
Reopening on OP's request

**Comment by Generalcamo** [comment]
!review

File: code/datums/outfits/event/outfit_nanotrasen.dm [hunk 1]
```diff
@@ -233,10 +247,16 @@
 	backpack_contents = list(
 		/obj/item/storage/box/survival/engineer = 1,
 		/obj/item/storage/box/zipties = 1,
-		/obj/item/clothing/head/helmet = 1
+		/obj/item/gun/energy/gun = 1,
+		/obj/item/clothing/head/helmet/security = 1
```
Comment (Generalcamo) [inline code/datums/outfits/event/outfit_nanotrasen.dm:251]:
```suggestion
		/obj/item/clothing/head/helmet = 1
```

**Comment by Generalcamo** [comment]
CCIA approved:
image

**Review by NonQueueingMatt — APPROVED** [review]

**Review by FluffyGhoster — APPROVED** [review]

**Pull request merged**